cheatsheets/Java_Security_Cheat_Sheet.md

```diff
@@ -598,20 +602,24 @@ class AesGcmSimple {
 
 </details>
 
-#### Symmetric example using built in JCA/JCE classes
+_**End of code snippet.**_
+
+#### Symmetric example using built-in JCA/JCE classes
 
-If you absolutely cannot use a separate library, it is still possible to use the built JCA/JCE classes but it is strongly recommended to have a cryptography expert review the full design and code.
+If you absolutely cannot use a separate library, it is still possible to use the built JCA/JCE classes but it is strongly recommended to have a cryptography expert review the full design and code, as even the most trivial error can severely weaken your encryption.
 
 The following code snippet shows an example of using AES-GCM to perform encryption/decryption of data.
 
-A few contraints/pitfalls with this code:
+A few constraints/pitfalls with this code:
 
 - It does not take into account key rotation or management which is a whole topic in itself.
 - It is important to use a different nonce for every encryption operation, even if the same key is used.
 - The key will need to be stored securely.

```

Comment:
Perhaps a link to the [Cryptographic Storage Cheat Sheet](https://cheatsheetseries.owasp.org/cheatsheets/Cryptographic_Storage_Cheat_Sheet.html) is in order here???

Reply:
That is linked at the very beginning